every_project/sidekiq.md

```diff
@@ -19,13 +21,13 @@ class HardWorker
 end
 ```
 
-### 2. Ensure Capistrano stops and starts Sidekiq on deploys
+### 3. Ensure Capistrano stops and starts Sidekiq on deploys
 1. Add `gem 'capistrano-sidekiq', '~> 0.20.0'` to your Gemfile in the development environment and run `bundle install`
 2. Add `require 'capistrano/sidekiq'` to your `Capfile`
 
 Now when you run cap deploy commands, they should stop and restart Sidekiq.
 
-### 3. To enable the admin panel, mount the Sidekiq web UI.
+### 4. To enable the admin panel, mount the Sidekiq web UI.
```

Comment:
The current ordering makes this sound optional.  I would make the statement declarative:
`Mount the Sidekiq web UI to enable the admin panel`